app/controllers/charges_controller.rb

```diff
@@ -0,0 +1,36 @@
+class ChargesController < ApplicationController
+  before_action :authenticate_user!
+
+  def new
+  end
+
+  def create
+    # Amount in cents
+    @amount = 1000
```

Comment:
If that's a constant, I'd rather put it outside any method (such as above or below `before_action`) as `SUBSCRIPTION_COST=10`.